packages/desktop/App.svelte

```diff
@@ -199,7 +199,7 @@
             {#if settings}
                 <Settings handleClose={() => (settings = false)} />
             {/if}
-            <ToastContainer classes="absolute right-5 bottom-5 w-100" />
+            <NotificationManager />
```

Comment:
nice name :wink:

Reply:
:wink:

Reply:
we lost a precious opportunity here to have called this component `Toaster` :trollface: